klippy/extras/save_variables.py

```diff
@@ -0,0 +1,62 @@
+# Get and Save arbitrary variables in the config file so that values can be kept
+# across restarts.
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
```

Comment:
Please shrink the one line description to just one line.

There needs to be a copyright statement present - see the other Klipper files for an example layout.

Reply:
Done